tests/system/data_sources/test_postgres.py

```diff
@@ -534,15 +556,16 @@ def test_schema_validation(cloud_sql):
     "data_validation.state_manager.StateManager.get_connection_config",
     new=mock_get_connection_config,
 )
-def test_schema_validation_core_types():
+def test_schema_validation_pg_types():
+    """Test schema validation on most PostgreSQL scalar data types."""
```

Comment:
@nj1973 Maybe we add to the comment that this test used to be the "core types" one? For reference purposes, since the others will keep using "core" for their methods

Reply:
Yep, good idea, I've updated the comments.